src/components/material/MaterialMainfestationItem.tsx

```diff
@@ -163,11 +168,18 @@ const MaterialMainfestationItem: FC<MaterialMainfestationItemProps> = ({
           <img src={ExpandIcon} alt="" />
         </div>
         {isOpen && (
-          <MaterialDetailsList
-            id={detailsId}
-            className="mt-24"
-            data={detailsListData}
-          />
+          <>
+            <MaterialDetailsList
+              id={detailsId}
+              className="mt-24"
+              data={detailsListData}
+            />
+            <CopyLink
+              label={t("copyLinkToEditionText")}
+              url={`${window.location.origin}${window.location.pathname}${window.location.search}#${manifestationId}`}
```

Comment:
Considering that you created a function to return the hash, it seems odd that you just grab `window.location` here. A `getCurrentUrlWithHash()` function perhaps?